src/NATS.Client.Core/NatsTlsOpts.cs

```diff
@@ -77,6 +77,8 @@ public sealed record NatsTlsOpts
     /// <summary>When true, skip remote certificate verification and accept any server certificate</summary>
     public bool InsecureSkipVerify { get; init; }
 
+    public bool CheckCertificateRevocation { get; init; }
```

Comment:
Should we change this to be 

```
public X509RevocationMode CertificateRevocationCheckMode {get; init;}
```

By making it a bool that flips between `NoCheck` and `Online`, we are not allowing the option for the caller to select `Online`